api/model.go

```diff
@@ -417,6 +417,8 @@ func volumeSchema(volume *client.Schema) {
 	volume.ResourceFields["replicas"] = replicas
 
 	recurringJobs := volume.ResourceFields["recurringJobs"]
+	recurringJobs.Create = true
```

Comment:
What are those two fields for?

Reply:
Make `recurringJobs` become configurable in volume create api

Reply:
Yeah, misread it